src/main/scala/fi/oph/koski/validation/KoskiValidator.scala

```diff
@@ -513,6 +513,7 @@ class KoskiValidator(
         :: Lukio2019OsasuoritusValidation.validate(suoritus, parent)
         :: Lukio2019VieraatKieletValidation.validate(suoritus, parent)
         :: Lukio2019ArvosanaValidation.validateOsasuoritus(suoritus)
+        :: validateLukioonValmistava2019(suoritus)
```

Comment:
Lukio2019-validointeja tehdessä oli muistaakseni puhetta, että alettaisiin muutenkin tehdä näitä spesifejä validointeja erilliseen objekteihin kuten nuo Lukio2019-validoinnit tosiaan on tehty, jotta KoskiValidator ei paisuisi enää enempää.